### DISCUSSION TITLE
War Russia agains Ukraine

### DISCUSSION THREAD

**Discussion by Stas0238** [community/community#12091] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, 
I know that this area is not for such reasons but I am a citizen of a country where Russia started war. I am from Ukraine.
I decided to write here to make you known that the war is real and it could touch anyone if it won't be stopped in Ukraine. If you could share any information about support Ukraine or for russian users to go on their squares and show their thoughts (they support Ukraine but their goverment decided this war and they should show their own thought - the citizens are the real goverment of any country). -> Check the video to see how real russians show their personality -> https://www.facebook.com/100002305693349/posts/4864505846969557/?d=n

Below I will put the text of our live chats from war news:
`_**I appeal to all Journalists from Civilized countries currently working in Ukraine!

 At the time of the terrible tragedy that is now being played out in our peaceful country, the role of the Media is extremely important.

 Ukrainian media are not broadcast in the Russian Federation.  But the leading Western media watch and read there.

 We desperately need your help to show that the Russian troops were deceived by Putin.

 They were not even told that they were going to attack Ukraine.  And they said that this is a continuation of the exercises.

 The Russian military often surrender without a fight.  They are absolutely disoriented and do not understand what they are doing in Ukraine.

 Please show the facts of the surrender of enemy soldiers.  This will help us demotivate the aggressor!

 This is VERY VERY important!

 This will be your contribution to stopping aggression and the third world war!**_`
I am not sure if you answer me positive and make anything but to make you known - if any other country decided to come inside your own and take your borders, cities, kill your friends, families - what would you do then ? 
We are independent country and we need to show that the truth is on our lips.

**Comment by vedantmgoyal9** [comment, marked as answer]
Please put this at: https://github.com/github/feedback/discussions/12042
